source/helpers/bisect.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'dom-chef';
+import cache from 'webext-storage-cache';
+import select from 'select-dom';
+
+import features from '../features';
+
+export default async function bisectFeatures(): Promise<Record<string, boolean>> {
+	const bisectedFeatures: string[] | undefined = await cache.get('bisect');
+	if (bisectedFeatures === undefined) {
+		return {};
+	}
+
+	const enabledFeatures = new Set(bisectedFeatures.slice(0, Math.ceil(bisectedFeatures.length / 2)));
+	if (bisectedFeatures.length === 0) {
+		createMessageBox(<div><p>Every feature has been disabled. Do you still see the change or issue?</p><p>Note: You can change page here, but don’t use multiple tabs.</p></div>);
+	} else if (bisectedFeatures.length === 1) {
+		createMessageBox(<p>The change or issue is caused by <a href={'https://github.com/sindresorhus/refined-github/blob/main/source/features/' + bisectedFeatures[0] + '.tsx'}><code>{bisectedFeatures[0]}</code></a>.</p>, false);
+	} else {
+		createMessageBox(<p>Can you see the change or issue? ({Math.ceil(Math.log2(bisectedFeatures.length))} steps remaining.)</p>);
+	}
+
+	return Object.fromEntries(features.list.map(feature => [`feature:${feature}`, enabledFeatures.has(feature)]));
+}
+
+function createMessageBox(message: Element, yesNoButtons = true): void {
+	document.body.append(
+		<div id="rgh-bisect-dialog" className="Box p-3" style={{position: 'fixed', bottom: 10, left: '50%', maxWidth: '600px', transform: 'translateX(-50%)'}}>
```

Comment:
Maybe I'd also add a generous red box-shadow so it appears to be on top of everything, because it kind of blends in.

Example `box-shadow: 0 0px 40px var(--color-text-danger);`

<img width="539" alt="Screen Shot 2021-04-05 at 02 19 26" src="https://user-images.githubusercontent.com/1402241/113519282-5e618280-95b5-11eb-8867-8b109b9a22ca.png">

Reply:
Or a lighter `var(--color-toast-shadow)` (which might be available as a class too)

<img width="505" alt="Screen Shot 2021-04-05 at 02 21 39" src="https://user-images.githubusercontent.com/1402241/113519335-b7c9b180-95b5-11eb-85d4-8d120efc9ee4.png">